.github/workflows/smoke-test.yml

```diff
@@ -35,6 +35,9 @@ jobs:
     - name: Run unit tests with pytest
       run: |
         python -m pytest tests
+    - name: Get latest pz-rail-base
+      run: |
+        pip install git+https://github.com/LSSTDESC/rail_base
```

Comment:
I think I'd prefer that we roll a new release of rail_base rather than grab from GitHub here, that seems more consistent with our practices in other repos.

Reply:
I agree, especially as we don't install any of the other packages from source.

Reply:
Okay! I'll take it out, just wanted to be mindful of talks about release practices but have no personal preference